cypress/integration/homeToCountries_spec.js

```diff
@@ -0,0 +1,7 @@
+describe('The Home Page To List', () => {
+    it('successfully loads & links to countries list', () => {
```

Comment:
Title should be more like, "redirect to countries" because you don't check whether on countries page there is everything rendered properly, you just checked if url is ok